tests/reporter/kubernetes.go

```diff
@@ -516,6 +527,100 @@ func (r *KubernetesReporter) logEvents(virtCli kubecli.KubevirtClient, since tim
 	fmt.Fprintln(f, string(j))
 }
 
+func (r *KubernetesReporter) dumpSRIOVInfo(virtCli kubecli.KubevirtClient) {
+	sriovOutputDir := filepath.Join(r.artifactsDir, "sriov")
+	if err := os.MkdirAll(sriovOutputDir, 0777); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create directory: %v\n", err)
+		return
+	}
+
+	r.logSRIOVNodeState(virtCli)
+	r.logSRIOVNodeNetworkPolicies(virtCli)
+	r.logNetworks(virtCli)
+	r.logOperatorConfigs(virtCli)
+
+	sriovPods, err := virtCli.CoreV1().Pods(sriovNamespace).List(metav1.ListOptions{})
```

Comment:
Seems that the events logging should be in a separate method, just like logNetworks/policies. Any reason for this to be asynchronise?